content/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services.md

```diff
@@ -73,6 +73,16 @@ In the following example, `StringLike` is used with a wildcard operator (`*`) to
 }
 ```
 
+The `sub` will not contain the branch name if you are using a workflow with an environment. Here, the `ref:refs/heads/branchName` will not work. You need to update the `sub` to contain the reference to the environment's name. Detailed information in [OIDC - Filtering for a specific branch](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect#filtering-for-a-specific-branch)
```

Comment:
Should this example be moved up to _before_ the previous one that shows wildcard condition for the `sub` claim? Because if customer defined wildcard condition then it will cover all cases, whether branch or environment are being used.